app/controllers/broadcast/items_controller.rb

```diff
@@ -3,6 +3,7 @@ class Broadcast::ItemsController < ApplicationController
   before_action :set_item, except: [:index, :new, :create, :setup_generation, :generate_script, :need_translation, :scripts, :view_script]
 
   def index
+    @language = Translations::Language[params[:lang] || 'en-US']
     @items = conditional_filter Broadcast::Item.active, originated_at: params[:originated_at], broadcast_municipality_id: params[:municipality],
                                                         source: params[:source], id: params[:id]
     @items = @items.includes(:municipality).order(originated_at: :desc).paginate page: params[:page], per_page: 100
```

Comment:
Call `includes(association_name)` to do a soft join without actually needing to do any filtering on the joined table.

`@items.includes(:municipality).includes(:translations)...`

Reply:
This appears to not have made much difference in the queries I see in the logs.